_translations/en.yml

```diff
@@ -31,12 +31,8 @@ en:
     title: "Bitcoin for Businesses - Bitcoin"
     pagetitle: "Bitcoin for Businesses"
     summary: "Bitcoin is a very secure and inexpensive way to handle payments."
-    lowfee: "The lowest fees out there"
-    lowfeetext: "Bitcoin's high cryptographic security allows it to process transactions in a very efficient and inexpensive way. You can make and receive payments using the Bitcoin network with almost no fees. In most cases, fees are not strictly required but they are recommended for faster confirmation of your transaction."
     fraud: "Protection against fraud"
     fraudtext: "Any business that accepts credit cards or PayPal knows the problem of payments that are later reversed. Chargeback frauds result in limited market reach and increased prices, which in turn penalizes customers. Bitcoin payments are irreversible and secure, meaning that the cost of fraud is no longer pushed onto the shoulders of the merchants."
-    international: "Fast international payments"
-    internationaltext: "Bitcoins can be transferred from Africa to Canada in 10 minutes. In fact, bitcoins never have any real physical location, so it is possible to transfer any amount anywhere with no limits, delays, or excessive fees. There are no intermediate banks to make you wait three business days."
```

Comment:
Sure, but you've been in #bitcoin-dev the same as me when random strangers start complaining about those one or two blocks a week that take 50 minutes or more.  An unexpected 40 minute or longer wait is a bad UX, so we should probably either provide an accurate expectation or not mention 10 minutes at all.

I agree that this isn't documentation, but good marketing sells the product as accurately as possible so that the customer gets exactly what they expected.  Barring that, it's better to promise less and deliver more.